v2/src/lib.rs

```diff
@@ -520,11 +515,12 @@ impl Storage for InfluxDbStorage {
         let db = get_db_name(self.config.clone())?;
 
         let start_timestamp = NaiveDateTime::UNIX_EPOCH;
-        let stop_timestamp = NaiveDateTime::from_timestamp_opt(
+        let stop_timestamp = chrono::DateTime::from_timestamp(
```

Comment:
`from_timestamp_opt`
👎Deprecated since `0.4.35`: use `DateTime::from_timestamp` instead
https://docs.rs/chrono/latest/chrono/naive/struct.NaiveDateTime.html#method.from_timestamp_opt

`chrono::DateTime::from_timestamp` returns a  `DateTime`